src/Config.js

```diff
@@ -236,6 +236,10 @@ export class Config {
     return this.customPages.passwordResetSuccess || `${this.publicServerURL}/apps/password_reset_success.html`;
   }
 
+  get parseFrameURL() {
+    return this.customPages.parseFrameURL || null;
```

Comment:
Rebased and will just return the value of `parseFrameURL` which will be `undefined` if, well, undefined.